source/_components/opentherm_gw.markdown

```diff
@@ -49,7 +49,12 @@ climate:
       required: false
       type: boolean
       default: false
+monitored_variables:
+  description: "A list of variables to expose as sensors."
+  required: false
+  type: list
 {% endconfiguration %}
```

Comment:
Add a `keys` key here and list all the variables.

Reply:
The full list is on the sensor docs page, with a reference just below the config option here. It will be quite a long list to add here, especially when binary sensors will be implemented as well. Keeping the list on the sensors (and later on binary sensors) page(s) will also help people keep track of what kind of sensor to expect.

Reply:
I don't think we have a precedence for not including the full config spec here.

Reply:
Done in b554915.